### PR TITLE
fix,traslate(2-1.5): Fix and traslate `Node properties: type, tag and contents` article

### PR DESCRIPTION
**Pull request by ltlaitoff**
Fixed translation errors

Synced `dom-class-hierarchy.svg` with original version and translated
Synchronized explanation of base classes and traslated

### CODE REVIEW

File: 2-ui/1-document/05-basic-dom-node-properties/article.md [hunk 1]
```diff
@@ -463,7 +486,7 @@ elem.innerHTML = elem.innerHTML + "..."
 </script>
 ```
 
-Найбільш стандартні атрибути HTML мають відповідну DOM властивість, і ми можемо отримати доступ до нєї.
+Найбільш стандартні атрибути HTML мають відповідну DOM-властивість, і ми можемо отримати доступ до нєї.
```
Comment (kester36) [inline 2-ui/1-document/05-basic-dom-node-properties/article.md:489]:
```suggestion
Найбільш стандартні атрибути HTML мають відповідну DOM-властивість, і ми можемо отримати доступ до неї.
```

File: 2-ui/1-document/05-basic-dom-node-properties/article.md [hunk 2]
```diff
@@ -476,10 +499,10 @@ elem.innerHTML = elem.innerHTML + "..."
 Основні властивості DOM вузла це:
 
 `nodeType`
-: Ми можемо використовувати цью властивість, щоб побачити, чи є вузол текстовим чи елементом. Вона має числове значення: `1` для елементів, `3` для текстових вузлів, а також кілька інших значень для інших типів вузлів. Лише для читання.
+: Використовується для визначення, чи є вузол текстовим чи елементним вузлом. Має числове значення: `1` для елементів, `3` для текстових вузлів та декілька інших значень для інших типів вузлів. Тільки для читання.
```
Comment (kester36) [inline 2-ui/1-document/05-basic-dom-node-properties/article.md:502]: Pay attention:
You've translated `Лише` to `Тільки` here, but not in the next row.
Maybe it is better to use the same word in both rows.

File: 2-ui/1-document/05-basic-dom-node-properties/article.md [hunk 3]
```diff
@@ -488,10 +511,10 @@ elem.innerHTML = elem.innerHTML + "..."
 : Повний HTML елемента. Операція запису в `elem.outerHTML` не змінює сам `elem`. Замість цього він замінюється новим HTML у зовнішньому контексті.
 
 `nodeValue/data`
-: Вміст неелементного вузла (тексту, коментаря). Ці дві властивості майже однакові, зазвичай ми використовуємо `data`. Можна змінювати.
+: Вміст не-елементного вузла (тексту, коментаря). Ці дві властивості майже однакові, зазвичай ми використовуємо `data`. Можна змінювати.
 
 `textContent`
-: Текст всередині елемента: HTML мінус всі `<теги>`. Написане в ній вставиться текстом всередину елемента, з усіма спеціальними символами та тегами, які обробляються точно так, як текст. Дозволяє безпечно вставити користувацький текст і захистити від небажаних HTML-вставок.
+: Текст всередині елемента: HTML мінус усі `<теги>`. Записуючи в нього, ми отримуємо текст всередині елемента, з усіма спеціальними символами та тегами, трактованими як текст. Це безпечний спосіб вставляти користувацький текст та захист від небажаних вставок HTML.
```
Comment (kester36) [inline 2-ui/1-document/05-basic-dom-node-properties/article.md:517]:
```suggestion
: Текст всередині елемента: HTML мінус усі `<теги>`. Записуючи в нього, ми отримуємо текст всередині елемента, з усіма спеціальними символами та тегами, трактованими як текст. Дозволяє безпечно вставляти користувацький текст, захищаючи від небажаних вставок HTML.
```

**Review by kester36 — CHANGES_REQUESTED** [review]

**Comment by ltlaitoff** [comment]
@kester36 Thanks for the review. It's really very useful for creating a good book in Ukrainian about JavaScript.

**Review by kester36 — APPROVED** [review]

**Review by stas-dolgachov — APPROVED** [review]

**Pull request merged**